squidb/src/com/yahoo/squidb/utility/VersionCode.java

```diff
@@ -18,6 +18,13 @@
     public static final VersionCode V3_7_4 = new VersionCode(3, 7, 4, 0); // Default on API 14, default minimum
```

Comment:
I think it makes sense to have use-case specific version codes that are more descriptively named, so that it's easier to understand where they might be used. I've deprecated all the constants in `VersionCode` and defined a new one for multi-row insert, i.e. `Insert.SQLITE_VERSION_MULTI_ROW_INSERT`. Want to give it another look?